app/src/main/java/com/infomaniak/drive/data/documentprovider/CloudStorageProvider.kt

```diff
@@ -258,17 +258,29 @@ class CloudStorageProvider : DocumentsProvider() {
         SentryLog.d(TAG, "openDocument(), id=$documentId, mode=$mode, signalIsCancelled: ${signal?.isCanceled}")
         val context = context ?: return null
 
+        fun getRemoteFile(localFile: File?, fileId: Int, driveId: Int): File? {
+            val userId = getUserId(documentId)
+            val okHttpClient = runBlocking { AccountUtils.getHttpClient(userId.toInt()) }
+            return ApiRepository.getFileDetails(localFile ?: File(id = fileId, driveId = driveId), okHttpClient).data
+        }
+
```

Comment:
```suggestion
    fun getRemoteFile(localFile: File?, fileId: Int, driveId: Int): File? {
         val okHttpClient = runBlocking { AccountUtils.getHttpClient(getUserId(documentId).toInt()) }
         return ApiRepository.getFileDetails(localFile ?: File(id = fileId, driveId = driveId), okHttpClient).data
    }
```
No need to create an extra variable for `userId`.

Reply:
Done